### DISCUSSION TITLE
can't show plots using matplotlib

### DISCUSSION THREAD

**Discussion by lenohard** [marimo-team/marimo#571] [Q&A] [answered]
Every time I execute a cell, which includes displaying a matplotlib plot, the result appears below the cell. However, the cell keeps running, and the loading indicator continues spinning. Consequently, I am unable to run other cells. After investigation, I discovered that my proxy application intercepts the http://[::]:52421 (random ports) request, causing this issue. Despite my efforts, I haven't found a solution to bypass this interception. (I attempted using http://localhost:52421, this time it resulted in the plot opening in another tab in my browser even when the proxy app was running.)

**Comment by mscolnick** [comment]
Can you include a snippet of your code?

You should end your code with `plt.gca()` to display it in the notebook:

```python
import matplotlib.pyplot as plt

plt.plot([-8, 8])
plt.ylabel("my variable")
plt.tight_layout()
plt.gca()
```

**Reply by lenohard** [reply, marked as answer]
It's already fixed after update during someday in March, thanks a lot, sorry for the delayed reply.
